tasks/dd_apm_logs_config.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Create Datadog logging directories
+  file:
+    path: "/etc/datadog-agent/conf.d/{{ item }}.d/"
+    state: directory
+    mode: 0755
+    owner: dd-agent
+    group: dd-agent
+  loop: "{{ sonar_log_names }}"
+  tags:
+    - datadog
+
+- name: Create Datadog log config files.
+  template:
+    src: datadog-logs-config.yml
+    dest: "/etc/datadog-agent/conf.d/{{ item }}.d/config.yaml"
+    mode: 0755
+    owner: dd-agent
+    group: dd-agent
+  loop: "{{ sonar_log_names }}"
+  notify: restart datadog-agent
+  tags:
+    - datadog
+
+- name: Create Datadog APM directory
+  file:
+    path: /opt/datadog-agent
```

Comment:
Why not `/opt/datadog-java-agent` like my previous comment and where this is stored on other servers? Any reason not to remain consistent?

Reply:
Sorry about that.